src/components/VFilters/VSearchGridFilter.vue

```diff
@@ -98,16 +105,47 @@ export default {
       }
     )
 
+    /**
+     * Find the last focusable element in VSearchGridFilter to add a 'Tab' keydown event
+     * handler to it.
+     * We could actually hard-code this because 'searchBy' is always the last now.
+     */
+    const lastFocusableElement = ref<HTMLElement>(null)
+    const updateLastFocusable = () => {
+      const focusable = getFocusableElements(filtersFormRef.value)
+      lastFocusableElement.value = focusable[focusable.length - 1]
+    }
+    onMounted(() => {
+      updateLastFocusable()
+    })
+    watch(filterTypes, () => updateLastFocusable())
+    /**
+     * When the user presses 'Tab' on the last focusable element, we need to
+     * move focus to the first focusable element in main.
+     * @param event
+     */
```

Comment:
So in addition to the last focusable element... we also need the _first_ focusable element to shift focus to the filter button when `shift + tab` is pressed.

Currently if you tab into the filter sidebar from the filter button (works great, amazing!) but then press <kbd>Shift</kbd> + <kbd>Tab</kbd> you end up on the 'back to top button'